shape_msgs/MeshTriangle[] triangles
geometry_msgs/Point32[] vertices
geometry_msgs/Vector3[] vertex_normals
std_msgs/ColorRGBA[] vertex_colors
